packages/amplify-cli/src/domain/amplify-toolkit.ts

```diff
@@ -113,10 +115,22 @@ export class AmplifyToolkit {
     this._getPlugin = this._getPlugin || require(path.join(this._amplifyHelpersDirPath, 'get-plugin')).getPlugin;
     return this._getPlugin;
   }
-  get getCategoryPlugins(): any {
-    this._getCategoryPlugins =
-      this._getCategoryPlugins || require(path.join(this._amplifyHelpersDirPath, 'get-category-plugins')).getCategoryPlugins;
-    return this._getCategoryPlugins;
+  get getCategoryPluginInfo(): any {
+    this._getCategoryPluginInfo =
+      this._getCategoryPluginInfo || require(path.join(this._amplifyHelpersDirPath, 'get-category-pluginInfo')).getCategoryPluginInfo;
+    return this._getCategoryPluginInfo;
+  }
+  get getConsoleHostingPluginInfo(): any {
```

Comment:
Console hosting is not a normal plugin from the requirement, like its insertion into the status command, it in some sense got to the plugin platform itself. Just thought it might be needed in other scenarios.